### ISSUE TITLE
@walletconnect/universal-provider 2.7.2 compile failed, ask for 

### ISSUE THREAD

**Issue by windatang** [WalletConnect/walletconnect-monorepo#2304]
**Describe the bug**
when dependency go to @walletconnect/universal-provider ^2.7.2
react build package failed  compile failed, ask for install react-native
```
/vercel/path0/node_modules/@walletconnect/universal-provider/dist/index.es.js
Cannot find module: 'react-native'. Make sure this package is installed.
```

 

**SDK Version (if relevant)** @walletconnect/universal-provider 
 - Client: React Dom
 - Version 2.7.2

**To Reproduce**
1. yarn add @walletconnect/ethereum-provider 2.7.0
2. auto dependency will go to @walletconnect/universal-provider  2.7.2 
3. npm build (react-app-rewired build) 
4. compile failed 
```
/vercel/path0/node_modules/@walletconnect/universal-provider/dist/index.es.js
Cannot find module: 'react-native'. Make sure this package is installed.
```
```yarn.lock
"@walletconnect/ethereum-provider@^2.7.0":
  version "2.7.2"
  resolved "https://registry.yarnpkg.com/@walletconnect/ethereum-provider/-/ethereum-provider-2.7.2.tgz#261c546883131e15faa4db6dc3ba020c6388d5af"
  integrity sha512-bvmutLrKKLlQ1WxKCvvX5p5YVox1D1f3Enp6hzAiZf4taN+n/M5rmwfAcLgLhp4cTAUDhl3zgtZErzDyJnvGvQ==
  dependencies:
    "@walletconnect/jsonrpc-http-connection" "^1.0.4"
    "@walletconnect/jsonrpc-provider" "^1.0.11"
    "@walletconnect/jsonrpc-types" "^1.0.2"
    "@walletconnect/jsonrpc-utils" "^1.0.7"
    "@walletconnect/sign-client" "2.7.2"
    "@walletconnect/types" "2.7.2"
    "@walletconnect/universal-provider" "2.7.2"
    "@walletconnect/utils" "2.7.2"
    events "^3.3.0"
```

**Comment by chosta** [comment]
Bumping this. I have the exact same issue.
Installing react-native is not an option.

**Comment by opeoyeleke** [comment]
I have the same issue . Any workaround or temp fix yet?

**Comment by opeoyeleke** [comment]
Editing yarn.lock and changing the versions of all "@walletconnect/ethereum-provider" instances from 2.7.2 to 2.7.0 worked for me

**Comment by ganchoradkov** [comment]
Hey. we've just released `2.7.4` and the issue should be fixed
